docs/source/overview/getting_started.rst

```diff
@@ -18,6 +18,34 @@ Running from Source
 There are some examples of how to write queries in the `github examples
 directory <https://github.com/palantir/atlasdb/tree/develop/examples>`__.
 
+Running Tests Locally
+=====================
+Running atlas tests requires ``docker`` and ``docker-compose`` to be installed on the machine where they are running. Instructions for different operating systems can be found `here <https://docs.docker.com/engine/installation/>`__
+
+Instructions for MacOS
+----------------------
+Mac users will need to install `docker-machine` and a virtualization environment as well to be able to run docker. Virtualbox is used in this example. Simplified set-up instructions are as follows:
+
+1. Install ``docker-machine``, ``docker`` and ``docker-compose``
+   
+   .. code:: bash
+   
+      brew install docker docker-machine docker-compose
+
+2. Set up a ``docker-machine`` for Virtualbox (named ``atlas`` in the example, name can be chosen freely by the user)
```

Comment:
Minor nit: is it best practice to have different docker-machines for each project?  I found it easier to just have one `default` machine for everything.